cellprofiler/modules/graytocolor.py

```diff
@@ -76,94 +81,122 @@ def create_settings(self):
-                value=1, minval=0, doc='''
-            <i>(Used only if %(SCHEME_RGB)s is selected as the color scheme)</i><br>
-            Enter the relative weight for the green image. If all relative weights are equal, all three
-            colors contribute equally in the final image. To weight colors relative to each other,
-            increase or decrease the relative weights.''' % globals())
+                value=1, minval=0, doc='''\
+*(Used only if "%(SCHEME_RGB)s" is selected as the color scheme)*
+
+Enter the relative weight for the green image. If all relative weights
+are equal, all three colors contribute equally in the final image. To
+weight colors relative to each other, increase or decrease the relative
+weights.
+''' % globals())
 
         self.blue_adjustment_factor = cps.Float(
                 "Relative weight for the blue image",
-                value=1, minval=0, doc='''
-            <i>(Used only if %(SCHEME_RGB)s is selected as the color scheme)</i><br>
-            Enter the relative weight for the blue image. If all relative weights are equal, all three
-            colors contribute equally in the final image. To weight colors relative to each other,
-            increase or decrease the relative weights.''' % globals())
+                value=1, minval=0, doc='''\
+*(Used only if "%(SCHEME_RGB)s" is selected as the color scheme)*
+
+Enter the relative weight for the blue image. If all relative weights
+are equal, all three colors contribute equally in the final image. To
+weight colors relative to each other, increase or decrease the relative
+weights.
+''' % globals())
         # # # # # # # # # # # # # #
         #
         # CYMK settings
         #
         # # # # # # # # # # # # # #
         self.cyan_image_name = cps.ImageNameSubscriber(
                 "Select the image to be colored cyan", can_be_blank=True,
-                blank_text=LEAVE_THIS_BLACK)
+                blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image to be displayed in cyan""")
 
         self.magenta_image_name = cps.ImageNameSubscriber(
                 "Select the image to be colored magenta", can_be_blank=True,
-                blank_text=LEAVE_THIS_BLACK)
+                blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image to be displayed in magenta""")
 
         self.yellow_image_name = cps.ImageNameSubscriber(
                 "Select the image to be colored yellow", can_be_blank=True,
-                blank_text=LEAVE_THIS_BLACK)
+                blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image to be displayed in yellow""")
 
         self.gray_image_name = cps.ImageNameSubscriber(
                 "Select the image that determines brightness", can_be_blank=True,
-                blank_text=LEAVE_THIS_BLACK)
+                blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image that will determine each pixel's brightness""")
```

Comment:
Please add `*(Used only if "%(SCHEME_CMYK)s" is selected as the color scheme)*` plus a newline at the beginning of the doc string.  Sorry, should have thought of this before!